precompiles/staking/method_get_rewards_test.go

```diff
@@ -0,0 +1,101 @@
+package staking
+
+import (
+	"math/big"
+	"math/rand"
+	"testing"
+
+	"cosmossdk.io/math"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/stretchr/testify/require"
+	precompiletypes "github.com/zeta-chain/node/precompiles/types"
+	"github.com/zeta-chain/node/testutil/sample"
+)
+
+func Test_GetRewards(t *testing.T) {
+	t.Run("should return empty rewards list to a non staker", func(t *testing.T) {
```

Comment:
Consider adding a test for invalid coins

Reply:
Added in 2b130fdc0f29a4d958d599d633c42f3987a6e7b5